src/Func.h

```diff
@@ -1465,7 +1465,16 @@ class Func {
      * generated schedules might break when the sizes of the dimensions are
      * very different from the estimates specified. These estimates are used
      * only by the auto scheduler if the function is a pipeline output. */
-    Func &estimate(Var var, Expr min, Expr extent);
+    Func &set_estimate(Var var, Expr min, Expr extent);
+
+    //HALIDE_ATTRIBUTE_DEPRECATED("Use set_estimate() instead")
```

Comment:
I assume this is just commented out to be sure you catch all the cases in the initial branch?

Reply:
This is a draft PR. :-)